backend/server.py

```
@@ -7874,6 +7874,123 @@ async def legacy_scheduler_toggle(
+
+@app.get("/agent/voice/status")
+async def voice_status_backend(user: dict = Depends(get_current_user)):
+    """Check if voice transcription is available."""
+    try:
+        from agent.voice import VoiceCommandInterface
+        vi = VoiceCommandInterface()
+        return {
+            "mic_available": vi.mic_available,
+            "whisper_url": bool(vi._whisper_url),
+        }
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)